autogpts/forge/forge/sdk/errors.py

```diff
@@ -11,7 +11,7 @@ class AgentException(Exception):
     message: str
 
     hint: Optional[str] = None
-    """A hint which can be passed to the LLM to reduce reoccurrence of this error"""
+    """A hint which can be passed to the LLM to reduce recurrence of this error"""
```

Comment:
The sentence flow is a bit off maybe we can say something like:

```suggestion
    """A hint that can be given to the language model to lessen the frequency of this error."""
```